sympy/functions/elementary/trigonometric.py

```diff
@@ -450,21 +469,17 @@ def fdiff(self, argindex=1):
     @classmethod
     def eval(cls, arg):
         from sympy.functions.special.polynomials import chebyshevt
+        from sympy.calculus.util import Limits
         if arg.is_Number:
             if arg is S.NaN:
                 return S.NaN
             elif arg is S.Zero:
                 return S.One
             elif arg is S.Infinity or arg is S.NegativeInfinity:
-                # In this cases, it is unclear if we should
-                # return S.NaN or leave un-evaluated.  One
-                # useful test case is how "limit(sin(x)/x,x,oo)"
-                # is handled.
-                # See test_sin_cos_with_infinity() an
-                # Test for issue 3308
-                # https://github.com/sympy/sympy/issues/5196
-                # For now, we return un-evaluated.
-                return
+                return Limits(-1, 1)
```

Comment:
I will add a note here, mentioning why it is better to return `Limits(-1, 1)` instead of returning `nan`.